index.js

```diff
@@ -1,10 +1,22 @@
 module.exports = app => {
   app.log('Yay, the app was loaded!')
- app.on('issue_comment', async context => {
-   // `context` extracts information from the event, which can be passed to
-   // GitHub API calls. This will return:
-   //   {owner: 'yourname', repo: 'yourrepo', number: 123, body: 'Hello World!}
-  context.log(context.payload.comment.body)
- })
-}
+  app.on('issue_comment', async context => {
+    // `context` extracts information from the event, which can be passed to
+    // GitHub API calls. This will return:
+    //   {owner: 'yourname', repo: 'yourrepo', number: 123, body: 'Hello World!}
+
+    const comparisonString = 'Can I work on this?'
+    const label = 'first-timers-only'
+    let check = false
+    const labelArray = context.payload.issue.labels
 
+    if (context.payload.comment.body === comparisonString) {
```

Comment:
I feel it'll be nice to abstract this out as a function.

`isClaimingIssue(comment.body)`

This way the logic of figuring that out can be separated out and updated without changing this piece of code.